src/modules/jcms_notifications/drush/jcms_notifications.drush.inc

```diff
@@ -95,12 +97,24 @@ function drush_jcms_notifications_article_import($lrp = FALSE) {
       drush_print(dt('Received message !message', ['!message' => var_export($message, TRUE)]));
       $id = $message->getId();
       if ($id) {
-        $articleVersions = $fetch_service->getArticleVersions($id);
-        $crud_service->crudArticle($articleVersions);
-        drush_print(dt('Processed message !message_id', ['!message_id' => $message->getMessageId()]));
-        $queue_service->deleteMessage($message);
-        drush_print(dt('Deleted from the queue message !message_id', ['!message_id' => $message->getMessageId()]));
-        $count++;
+        try {
+          $articleVersions = $fetch_service->getArticleVersions($id);
+          $crud_service->crudArticle($articleVersions);
+          drush_print(dt('Processed message !message_id', ['!message_id' => $message->getMessageId()]));
+        }
+        catch (Exception $e) {
+          if ($e instanceof RequestException) {
+            error_log($e);
+          }
+          else {
+            throw $e;
+          }
+        }
+        finally {
+          $queue_service->deleteMessage($message);
```

Comment:
the `finally` ensures that there can be no continuous retry of the same message, nice